Categories/Auras.lua

```diff
@@ -18,8 +18,6 @@ local newList, del = DogTag.newList, DogTag.del
 
 local currentAuras, currentDebuffTypes, currentAuraTimes, currentNumDebuffs
 
--- Parnic: support for cataclysm; Divine Intervention was removed
-local wow_400 = select(4, GetBuildInfo()) >= 40000
```

Comment:
Oops! :)

Reply:
That one is unused actually - figured I'd just remove it when I saw it.

Reply:
Yeah, that's what I meant by "oops" - I made that change at some point and didn't need it.